client/src/app/ui/modules/media-upload-content/components/media-upload-content/media-upload-content.component.html

```diff
@@ -19,13 +19,6 @@
                 ></os-list-search-selector>
             </mat-form-field>
         </div>
-
-        <div>
-            <span>{{ 'Upload to:' | translate }}</span>
-            &nbsp;
-            <i *ngIf="!directoryId">{{ 'Base folder' | translate }}</i>
-            <i *ngIf="directoryId">{{ getDirectoryTitle() }}</i>
-        </div>
```

Comment:
Also did some cleanup: This div (incorrectly) mirrored the selection directly above it, so I removed it and everything which was only there for it (e.g. `currentDirectory`).